frontend/src/metabase/ui/components/overlays/Menu/Menu.styled.tsx

```diff
@@ -13,10 +13,11 @@ export const getMenuOverrides = (): MantineThemeOverride["components"] => ({
         padding: "0.75rem !important",
         minWidth: "11.5rem",
         overflow: "auto",
-        background: theme.fn.themeColor("bg-white"),
+        background: "var(--mb-color-background)",
```

Comment:
Style the dropdown 
![image](https://github.com/user-attachments/assets/50160d5d-236c-4a71-91f5-ce389afad7be)